src/main/resources/db/migration/V1_10__Lagre_oppgave_oppdatert_timestamp.sql

```diff
@@ -0,0 +1,2 @@
+AlTER TABLE UTSATT_OPPGAVE
+    ADD COLUMN OPPDATERT TIMESTAMP NOT NULL;
```

Comment:
Denne sier at oppdatert MÅ være null. For gamle oppgaver så finnes det ikke noe tidspunkt og derfor må tillate NULL verdier.